tests/testthat/test-7-datalake.R

```diff
@@ -0,0 +1,91 @@
+
```

Comment:
I made some general comments on the tests in my other review.

Reply:
Fixed in https://github.com/omkarksa/AzureSMR/pull/1 will raise a pull request once this is merged into master